nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/hesai_decoder.hpp

```diff
@@ -36,6 +37,52 @@
 namespace nebula::drivers
 {
 
+struct HesaiDecodeFilteredInfo
+{
+  uint16_t distance_counter = 0;
+  uint16_t fov_counter = 0;
+  uint16_t timestamp_counter = 0;
+  float distance_start = 0;
+  float distance_end = 0;
+  float raw_azimuth_start = 0;
+  float raw_azimuth_end = 0;
+  std::uint32_t packet_timestamp_start = 0;
+  std::uint32_t packet_timestamp_end = 0;
+  NebulaPointCloud point_azimuth_start;
+  NebulaPointCloud point_azimuth_end;
+  NebulaPointCloud point_timestamp_start;
+  NebulaPointCloud point_timestamp_end;
```

Comment:
* Please rename from `start/end` to `min/max`.
* Please also add unit suffixes like `_ns` for nanoseconds, `_rad` for radians, `_m` for meters etc.
* `packet_timestamp_min/max` should probably have type `uint64_t` (`uint32_t` cannot represent absolute timestamps in nanoseconds)

Instead of `point_`, please rename to `cloud_` so that it is clear that those values are among the points that were not filtered.
I would suggest replacing `raw_` with `packet_` as well, so we have `packet_` (before filtering) vs. `cloud_` (after filtering).